README.rst

```diff
@@ -307,6 +307,10 @@ Requirements: Docker 19.03.2 or newer and Docker Compose 1.24.1 or newer.
 Release History
 ---------------
 
+Version 1.9.1
+
+- State compatibility with 3.6, 3.7 and 3.8
```

Comment:
Maybe also state this version drops compatibility with Django 1?

Reply:
Done